services/director-v2/src/simcore_service_director_v2/modules/resource_usage_client.py

```diff
@@ -8,17 +8,23 @@
 
 import httpx
 from fastapi import FastAPI
+from models_library.api_schemas_resource_usage_tracker.credit_transactions import (
+    WalletTotalCredits,
+)
 from models_library.api_schemas_resource_usage_tracker.pricing_plans import (
+    PricingUnitGet,
     ServicePricingPlanGet,
 )
 from models_library.products import ProductName
 from models_library.resource_tracker import (
+    PricingAndHardwareInfoTuple,
```

Comment:
WARNING: pleaes review carefully this PR. I cannot find here the changes of `models_library.resource_tracker` that you aded in https://github.com/ITISFoundation/osparc-simcore/pull/4878/commits/5b32ac0cc165d198f24216e2d39634566e7e130d

Reply:
I wonder why there are commits from me in your pR

Reply:
Answers:
1. The changes are there :+1: The only thing that complicated a bit is that in this PR I have moved the SQL queries from the webserver to the Postgres package (where we do not have models_libary, therefore after I call it, straight away I use `PricingAndHardwareInfoTuple` so it is clear what it is when working with it later. Is that ok?
2. Hmm, I merged master to my branch and needed to deal with some conflicts, maybe thats how you got in?